### PR TITLE
Generate bindings for Galaxy V3 APIs

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/5238
ref #5238

Required PR: https://github.com/pulp/pulpcore/pull/577

### CODE REVIEW

File: CHANGES/5238.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Generate bindings for Galaxy V3 APIs
```
Comment (bmbouter) [inline CHANGES/5238.misc]: This is a feature I think. We never did this before.

**Comment by fabricio-aguiar** [comment]
@bmbouter @cutwater does it help with galaxy_ng?

**Comment by bmbouter** [comment]
I generated the different schemas from master for pulp_ansible and pulpcore and then this PR and the correponding pulpcore PR. When I vimdiff I see some description fixed but not the addition of a description for the V3 API. Here's the diff I see from the openAPI schema from this PR versus master:

```
3203,3205d3202
<                         artifact:
<                           type: string
<                           description: Artifact file representing the physical content
3213a3211,3213
>                         artifact:
>                           type: string
>                           description: Artifact file representing the physical content
3486,3488d3485
<                   artifact:
<                     type: string
<                     description: Artifact file representing the physical content
3496a3494,3496
>                   artifact:
>                     type: string
>                     description: Artifact file representing the physical content
3657,3659d3656
<                   artifact:
<                     type: string
<                     description: Artifact file representing the physical content
3667a3665,3667
>                   artifact:
>                     type: string
>                     description: Artifact file representing the physical content
3968,3970d3967
<                         artifact:
<                           type: string
<                           description: Artifact file representing the physical content
3978a3976,3978
>                         artifact:
>                           type: string
>                           description: Artifact file representing the physical content
4019,4021d4018
<                   artifact:
<                     type: string
<                     description: Artifact file representing the physical content
4029a4027,4029
>                   artifact:
>                     type: string
>                     description: Artifact file representing the physical content
4058,4060d4057
<                   artifact:
<                     type: string
<                     description: Artifact file representing the physical content
4068a4066,4068
>                   artifact:
>                     type: string
>                     description: Artifact file representing the physical content
7306a7307,7308
>                     readOnly: true
>                     nullable: true
7316d7317
<                 - error
8027a8029,8030
>                     readOnly: true
>                     nullable: true
8037d8039
<                 - error

```

File: pulp_ansible/app/galaxy/views.py [hunk 2]
```diff
@@ -47,6 +49,8 @@ class RoleList(generics.ListAPIView):
     serializer_class = GalaxyRoleSerializer
     authentication_classes = []
     permission_classes = []
+    pulp_slug = False
+    pulp_full_tag = True
```
Comment (fabricio-aguiar) [inline pulp_ansible/app/galaxy/views.py]: added for helping with bindings

**Comment by fao89** [comment]
Closing as we are implementing OpenAPI v3

**Pull request closed without merging**